### PR TITLE
feat: move bare react native examples files to /rn-bare in /examples folder

### PR DESCRIPTION
**Pull request by iykazrji**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [x] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [x] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [x] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [x] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [x] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

## Massive PR
Most changes however are due to the moving of the react native examples files into the `rn-bare` project folder

---

## PR-Codex overview
This PR focuses on restructuring the `rn-signer` example by moving it to a new directory called `rn-bare` and updating relevant configurations, paths, and dependencies accordingly.

### Detailed summary
- Moved `account-kit/rn-signer/example` to `account-kit/rn-signer/examples/rn-bare`.
- Updated `package.json` name from `@account-kit/react-native-signer-example` to `@account-kit/react-native-signer-bare-example`.
- Adjusted paths in various configuration files (`babel.config.js`, `metro.config.js`, `react-native.config.js`).
- Changed version from `4.5.1` to `4.6.0` in `version.ts`.
- Updated CI workflows to reflect new paths for Gradle and CocoaPods.
- Deleted `.bundle/config` file.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by iykazrji** [comment]
* **#1187** <a href="https://app.graphite.dev/github/pr/alchemyplatform/aa-sdk/1187?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/alchemyplatform/aa-sdk/1187?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: account-kit/rn-signer/.github/workflows/ci.yml
Comment (moldy530) [inline account-kit/rn-signer/.github/workflows/ci.yml:1]: this file probably shouldn't have changed (it's also just here for a reference for us to use later

**Review by moldy530 — COMMENTED** [review]
Ah sorry this was on me...

the `rn-signer/example` folder should be untouched as we can use it to test the code as we build it (it's meant to be an internal example and is also used to validate the android and iOS builds)

we should add 2 new examples to the root `examples` folder:
1. `react-native-bare` -> uses the signer and account-kit/sca and account-kit/infra like https://github.com/alchemyplatform/aa-sdk-rn-expo does to generate a LA address. 
2. `react-native-expo` -> basically a migration of https://github.com/alchemyplatform/aa-sdk-rn-expo into the monorepo

Both of these shouldn't need to reference local node_modules if possible and just have all packages installed as part of their dependencies

**Comment by iykazrji** [comment]
Ah gotcha... I'd go ahead and close this one now

**Pull request closed without merging**